mediagrains/grain.py

```diff
@@ -36,12 +36,12 @@
 
 if __name__ == "__main__":  # pragma: no cover
     from uuid import uuid1, uuid5
-    from .grains import Grain
+    from .grains import GrainFactory
 
     src_id = uuid1()
     flow_id = uuid5(src_id, "flow_id:test_flow")
 
-    grain1 = Grain(src_id=src_id, flow_id=flow_id)
-    grain2 = Grain(grain1.meta)
+    grain1 = GrainFactory(src_id=src_id, flow_id=flow_id)
+    # grain2 = Grain(grain1.meta)
```

Comment:
NIT: Are these commented lines intentional? If possible, I'd either re-instate them or remove them entirely. If they need to stay, could you add a comment explaining whats going on.